CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## [2.0](https://github.com/arborx/arborx/tree/v2.0) (2025-04-22)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v1.7...v2.0)
+
+**New features:**
+- New interfaces for search indexes (APIv2) [\#1155](https://github.com/arborx/ArborX/pull/1155), [\#1157](https://github.com/arborx/ArborX/pull/1157), [\#1161](https://github.com/arborx/ArborX/pull/1161)
+- Hyper-dimensional geometries  [\#1100](https://github.com/arborx/ArborX/pull/1100), [\#1148](https://github.com/arborx/ArborX/pull/1148), [\#1146](https://github.com/arborx/ArborX/pull/1146), [\#1154](https://github.com/arborx/ArborX/pull/1154)
+
+**Build Changes:**
+- Require C++20 [\#1215](https://github.com/arborx/ArborX/pull/1215)
+- Require Kokkos 4.5.00 [\#1233](https://github.com/arborx/ArborX/pull/1233)
+- Require CMake 3.22 [\#1188](https://github.com/arborx/ArborX/pull/1188)
+- Update compiler requirements for C++20 [\#1215](https://github.com/arborx/ArborX/pull/1215)
```

Comment:
```suggestion
```

Reply:
I agree with you. But I don't know how to indicate to users to check what compiler requirements are.

Reply:
Maybe consolidate into a single entry "Require C++20 and bump CMake minimum requirement"